packages/oscal-react-library/docs/adding-editing.md

```diff
@@ -0,0 +1,28 @@
+# Adding editing to components
+
+## Text field editing
+
+Currently our editing functionality is done with `OSCALEditableTextField`
+& its helper component `OSCALEditableFieldAction`.
+
+### Important props
+
+- `fieldName`: Name of the field currently being editing.[^1]
+- `canEdit`: A boolean passed from the parent component on whether to be
+in Viewing or Editing mode. This will determine how `OSCALEditableTextField` is displayed.
+- `editedValue`: The value of the object being changed via the REST api call.
+This is **not required** if it is the same as `value`.[^2]
+- `editedValueId`: The id of the `editedValue`. Only required if `editedValue` is set.
+- `onFieldSave`: Currently this is the `handleFieldSave()` function passed down
+from the `OSCALLoader`. All of the funcionality of editing require this function.
+- `partialRestData`: A partial representation of the json object you are patching.
+- `value`: The value of the field that is changed on the front end.
+
+### Example PRs
+
+- [Editing backmatter resources](https://github.com/EasyDynamics/oscal-react-library/pull/768)
+
+[^1]: For example, "title" or "description".
+[^2]: This is only included if editing a value would update a list of objects.
+For example, editing a backmatter resource title would have an
+`editedValue` of `backmatter.resources` but a value of `resource.title`.
```

Comment:
Do you mind breaking this into a separate branch/PR. I have thoughts here but I don't want it to hold up review on the task itself.

Reply:
@kylelaker After addressing your comments below, I do not see much value in this file anymore. I think the interface documentation does a better job at describing these parameters. 

It has been deleted and, unless you disagree, I don't think there is much reason to make it into a new pr.